string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,20 @@
+/* exported capitalizeWord */
+function capitalizeWord(word) {
+  var result = word.charAt(0).toUpperCase() + word.slice(1).toLowerCase();
+  if (result.toLocaleLowerCase() === 'javascript') {
+    result = result.substring(0, result.indexOf('s')) + result.charAt(4).toUpperCase() + result.slice(5).toLowerCase();
```

Comment:
With very special cases like this, you don't need to assemble the return value yourself, you can just return the intended value directly aka `return "JavaScript";`